frontend/src/scenes/surveys/surveyLogic.tsx

```diff
@@ -405,10 +416,19 @@ export const surveyLogic = kea<surveyLogicType>([
         launchSurveySuccess: ({ survey }) => {
             lemonToast.success(<>Survey {survey.name} launched</>)
             actions.loadSurveys()
+            if (!values.currentTeam?.surveys_opt_in) {
+                actions.setSurveysOptIn(true)
+            }
```

Comment:
because this happens after the survey is launched, the banner flashes for a moment. From your video:

<img width="1254" alt="image" src="https://github.com/PostHog/posthog/assets/18598166/a287dad9-2b4e-4e3e-abfd-0a3386171277">

Maybe do this at the beginning of launchSurvey?